.circleci/config.yml

```diff
@@ -70,6 +90,142 @@ jobs:
           only_for_branches: master
       - store_artifacts:
           path: /tmp/tap-mongodb
+  build_mongo_4_4:
+    executor: tap_tester_mongo_4_4
+    steps:
+      - checkout
+      - run:
+          name: 'Install Dockerize'
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
```

Comment:
This would be much more readable with the multi line run block

Reply:
Agreed, moved all three of these to multi line format.